documentation-website/src/components/sponsor.scss

```diff
@@ -1,12 +1,6 @@
 .sponsor {
   display: grid;
-  grid-template-columns: 1fr;
-
-  img {
-    place-self: center;
-  }
-
-  h2 {
-    text-align: left;
-  }
+  justify-items: center;
+  align-items: center;
+  row-gap: 1ch;
```

Comment:
don't we have a single row here?

Reply:
> don't we have a single row here?

Do you mean Row as in Grids?

Reply:
forget it I'm misremembering the mobile layout